src/v/kafka/protocol/schemata/offset_fetch_request.json

```diff
@@ -28,7 +28,9 @@
   // Version 3, 4, and 5 are the same as version 2.
   //
   // Version 6 is the first flexible version.
-  "validVersions": "0-6",
+  //
```

Comment:
```
kafka/protocol: Upgrade offset_fetch_request to version 7
Added:
* RequireStable
```

Please record from which version of kafka you retrieved the json file. I noticed that for txn offset request the json matched from kafka 2.7 but this one doesn't match up to kafka 2.8, which is separately concerning, but we should at least record this to help anyone in the future trying to figure out the lineage of these files.

Reply:
I got this json from kafka pe for this kip. I do not know, how to understand kafka version...

Reply:
@dotnwat It seems it matches 2.5.0, where did you see otherwise?

```
cd kafka
git checkout 2.5.0
less ./clients/src/main/resources/common/message/OffsetFetchRequest.json
```

```
  ...
  "validVersions": "0-7",
  "flexibleVersions": "6+",
  ...
```

Reply:
Thanks @rystsov! I realized I was in the middle of an interactive rebase of this PR when I checked with diff. So the commit that updated it hadn't been applied 🤦